packages/layout/src/components/buttons/buttons.module.scss

```diff
@@ -1,5 +1,3 @@
-@import '@tpr/theming/lib/resets.module.scss';
-
 .removepadding {
 	padding-left: 0px !important;
```

Comment:
* You don't need units (px) for 0, which saves a couple of bytes
* [`!important` should normally be avoided](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception)
* This is another example of a class named after what it does at the moment, rather than the context in which it's applied

Reply:
renamed to `.arrowButton` as well as `px !important`